src/lib/preferences/preferences.cpp

```diff
@@ -214,7 +214,13 @@ Preferences::Preferences(BrowserWindow* window)
     //APPEREANCE
     settings.beginGroup("Browser-View-Settings");
     ui->showStatusbar->setChecked(settings.value("showStatusBar", false).toBool());
+    // NOTE: instantBookmarksToolbar and showBookmarksToolbar cannot be both enabled at the same time
```

Comment:
Here's an alternate mockup of the same UI logic but uses more screen real estate and gets rid of the cool name of "instant bookmarks toolbar":
![alternate](https://cloud.githubusercontent.com/assets/114709/17631871/116c2f14-6083-11e6-8312-958d9b64160e.png)